libraries/rush-lib/src/logic/base/BaseInstallManager.ts

```diff
@@ -126,15 +127,14 @@ export interface IInstallManagerOptions {
  * This class implements common logic between "rush install" and "rush update".
  */
 export abstract class BaseInstallManager {
-  private _rushConfiguration: RushConfiguration;
-  private _rushGlobalFolder: RushGlobalFolder;
-  private _commonTempInstallFlag: LastInstallFlag;
-  private _commonTempLinkFlag: LastLinkFlag;
```

Comment:
Was the link flag not getting used at all? IIRC I left it in-place for legacy rush installs. But if it's no longer cared about, I have no issue removing it.

Reply:
It's getting used. I'm just marking it as `readonly` here.